superset-frontend/src/pages/ExecutionLogList/index.tsx

```diff
@@ -119,7 +119,7 @@ function ExecutionLog({
             original: { scheduled_dttm: scheduledDttm },
           },
         }: any) =>
```

Comment:
Can we now use a better type than `any`?

Reply:
For this one i think keeping any is ok still